index.d.ts

```diff
@@ -20,10 +20,33 @@ export type Options = {
 	*/
 	readonly devToolsMode?:
 	| 'undocked'
+	| 'left'
 	| 'right'
 	| 'bottom'
 	| 'previous'
 	| 'detach';
+
+	/**
+	Specify customized options for each window.
+
+	@param window Window to apply the filter or new options to
```

Comment:
Incorrect use of `@param`. It's describing `windowSelector`, but the parameter is on the given callback function. Better to just not use the tags. Same with `@returns`.

You also have some incorrect indentation.

Reply:
Please suggest a the desired comments, because I can't read minds, specially when the comments are not following any standard nor have any automatic checks on CICD phase.
I can adjust the PR with that, but it's not worth to keep trying randomly until you like it.

Reply:
What you did with the docs looks good. I'm pointing out a technical issue. Your use of `@param` here is technically wrong, as `@param` is meant for an immediate function parameter, not parameters in a provided callback function. Same with `@returns`.

Reply:
So just, describing it with words would be ok for you?

> /**
> Specify customized options for each window.
> 
> The accepted function receives the window to apply the filter or new options to.
> It must return one of these values:
>   - `true`: To enable `debug` with the global options for the given `window`.
>   - `false`: Disable `debug` for the given `window` (same as returning `{ isEnabled: false }`).
>   - `Partial<Options>`: Object to extend and/or override the global options just for the given `window`.
> 
> @default `() => true`
> @example
> ```
> import debug from 'electron-debug';
> debug({
>   windowSelector: (window) => window.title !== 'Debug tools',
> });
> ```
> */

Reply:
👍